lib/modules/manager/gradle/artifacts.ts

```diff
@@ -157,6 +157,11 @@ export async function updateArtifacts({
       cmd += ` --update-locks ${updatedDepNames.map(quote).join(',')}`;
     }
 
+    // `./gradlew :dependencies` command can output huge text due to `:dependencies`
+    // that renders dependency graphs. Given the output can exceed `ExecOptions.maxBuffer` size,
+    // drop stdout from the command.
+    cmd += ' > /dev/null';
```

Comment:
would Gradle log errors to stderr?

Reply:
Yes, Gradle still leaves error messages to `stderr`.